postgresql-persistence/src/main/java/com/netflix/conductor/aurora/AuroraQueueDAO.java

```diff
@@ -33,8 +34,18 @@ public AuroraQueueDAO(DataSource dataSource, ObjectMapper mapper, Configuration
 		this.config = config;
 		loadQueues();
 
-		Executors.newSingleThreadScheduledExecutor()
-			.scheduleWithFixedDelay(this::processAllUnacks, UNACK_SCHEDULE_MS, UNACK_SCHEDULE_MS, TimeUnit.MILLISECONDS);
+		ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor();
+		executorService.scheduleWithFixedDelay(this::processAllUnacks, UNACK_SCHEDULE_MS, UNACK_SCHEDULE_MS, TimeUnit.MILLISECONDS);
+
+		Runtime.getRuntime().addShutdownHook(new Thread(() -> {
```

Comment:
Maybe we could consider creating of some Utils method like shutdownExecutor(executor, message) because we have similar code in 7 places